src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -182,7 +245,7 @@ public void run() {
                 // @checkstyle MagicNumber (1 line)
                 final byte[] buffer = new byte[4096];
                 for (int count = input.read(buffer); count != -1;
-                    count = input.read(buffer)) {
+                     count = input.read(buffer)) {
```

Comment:
@alexey-krylov the indentation step should be four spaces. Qulice check such cases from `0.15`
see https://github.com/teamed/qulice/issues/527
please remove the space